src/Pages/Quiz/quiz.jsx

```diff
@@ -0,0 +1,113 @@
+import { Link } from "react-router-dom";
+import { Navigation } from "../../components/Components";
+import { useQuiz } from "../../Context/quiz-context";
+
+function Quiz() {
+  const {
+    state: { index, ques,selectedOption,options,
+
+      selected },
+    dispatch,
+
+  } = useQuiz();
+
+
```

Comment:
you can remove extra space,space of 1 line is ok but ,you should avoid multiple line spacing